libecho/src/echo.cpp

```diff
@@ -1,20 +1,45 @@
 #include "echo.h"
-#include <iostream>
+#include "AudioOutput.h"
+#include "AudioFormatFactory.h"
+#include "AudioInput.h"
 
+#include <QCoreApplication>
+#include <QDebug>
 
-void echo::send(const std::vector<uint8_t>& buffer) {
-    std::cout << "Sending: ";
-    for (auto c : buffer) {
-        std::cout << c;
+
+void echo::send(const std::vector<char>& buffer) {
+    static AudioOutput audio(AudioFormatFactory::getDefaultOutputFormat());
+
+    /* Example transmission */
+    const size_t atOnce = 260; // (bitrate / notify_interval) + eps;
```

Comment:
It would be nice to have that formula ((bitrate / notify_interval) + eps) and why it is important documented somewhere.

Reply:
Fell free to work on it, it's not really my part. I just did something that seemed sensible to me, and only to check if my code works, it's not code that is supposed to stay here.